spec/index.html

```diff
@@ -2884,6 +2884,17 @@ <h3>Dataset Poisoning</h3>
     </p>
   </section>
 
+  <section id="insecure-hashing-algorithms" class="informative">
+    <h3>Insecure Hashing Algorithms</h3>
```

Comment:
```suggestion
  <section id="insecure-hash-algorithms" class="informative">
    <h3>Insecure Hash Algorithms</h3>
```
in line with @TallTed 's suggestion.